src/test/java/com/Bai18.java

```diff
@@ -0,0 +1,57 @@
+package com;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.utils.BasicTest;
+
+public class Bai18 extends BasicTest {
+    @Test()
+    public void loginTest() throws Exception {
+        // Launch website
+        String url = "https://bantheme.xyz/hathanhauto/tai-khoan/";
+        driver.get(url);
+
+        // Log in and go to home page
+        WebElement inputEmail = driver.findElement(By.xpath("//input[@id='username']"));
+        inputEmail.clear();
+        inputEmail.sendKeys("trangtq.9593@gmail.com");
+        WebElement inputPassword = driver.findElement(By.xpath("//input[@id='password']"));
+        inputPassword.clear();
+        inputPassword.sendKeys("Quynhtrang");
+        WebElement btnLogin = driver.findElement(By.xpath("//button[@name='login']"));
+        btnLogin.click();
+        String homepage = "https://bantheme.xyz/hathanhauto/";
+        driver.get(homepage);
+        String originalWindow = driver.getWindowHandle();
+
+        //Open New Tab
+        JavascriptExecutor jse = (JavascriptExecutor)driver;
+        jse.executeScript("window.open()");
+
+        //Loop through until we find a new window handle
+        for (String windowHandle : driver.getWindowHandles()) {
```

Comment:
cach loop nay cung hay day' :D

Ma trong bai tap thuc ra co moi 2 tab, nen em dung index 0 voi 1 cho nhanh luon cung dc.

Trong thuc te a cung it khi thay co testcase nao co nhieu tab : )

Anw, good work